src/main/java/org/json/JSONParserConfiguration.java

```diff
@@ -4,11 +4,32 @@
  * Configuration object for the JSON parser. The configuration is immutable.
  */
 public class JSONParserConfiguration extends ParserConfiguration {
+
+    /** Original Configuration of the JSON Parser. */
+    public static final JSONParserConfiguration ORIGINAL
+        = new JSONParserConfiguration();
+
+    /** Original configuration of the JSON Parser except that values are kept as strings. */
+    public static final JSONParserConfiguration KEEP_STRINGS
+        = new JSONParserConfiguration().withKeepStrings(true);
+
     /**
      * Used to indicate whether to overwrite duplicate key or not.
      */
     private boolean overwriteDuplicateKey;
 
+    /**
+     * This flag, when set to true, instructs the parser to throw a JSONException if it encounters an invalid character
+     * immediately following the final ']' character in the input. This is useful for ensuring strict adherence to the
+     * JSON syntax, as any characters after the final closing bracket of a JSON array are considered invalid.
+     */
+    private boolean strictMode;
+
+    /**
+     * Allows Single Quotes when strictMode is true. Has no effect if strictMode is false.
+     */
+    private boolean allowSingleQuotes;
```

Comment:
There should not be a separate flag for allowSingleQuotes. Strict mode means that all key and value strings must be in double quotes, and non-blank unparseable chars at the end of the doc are not allowed.

Reply:
Ok. I will remove it.